udfs/community/kll_sketch_float64.sqlx

```diff
@@ -0,0 +1,170 @@
+config { hasOutput: true }
+/*
+ * Copyright 2021 Google LLC
```

Comment:
```suggestion
 * Copyright 2024 Google LLC
```

Reply:
Update all other new files to be 2024 as well

Reply:
Updated